release-notes/3.0/preview/api-diff/preview4/3.0-preview4_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,44 @@
+# System.Runtime.CompilerServices
+
+``` diff
+ namespace System.Runtime.CompilerServices {
++    public static class Unsafe {
```

Comment:
cc @ericstj, now this is in box. Doesn't look like any APIs changed from the NuGet package, but just to verify.

Reply:
Lgtm